test/guessTheNumber_test.js

```diff
@@ -0,0 +1,8 @@
+// var expect = require('chai')
```

Comment:
what tests are you going to utilize? i think it'll be hard to create a test due to the randomness (like you can't determine what the answer will be to test against)